activesupport/lib/active_support/message_verifier.rb

```diff
@@ -128,9 +138,9 @@ class InvalidSignature < StandardError; end
 
     cattr_accessor :default_message_verifier_serializer, instance_accessor: false, default: :marshal
 
-    def initialize(secret, digest: nil, serializer: nil, url_safe: false)
+    def initialize(secret, digest: nil, **codec_options)
```

Comment:
I know that we already mention the `:serializer` and `:url_safe` options in the docs, but they aren't very easy to scan, in my opinion.  If we remove `:serializer` and `:url_safe` from the method signature, then perhaps we should add an `=== Options` section to the documentation to increase their visibility.  (It could replace the `=== Generating URL-safe strings` and `=== Backwards compatibility` sections.)

If you decide to go that way, I think a dictionary-style list makes it easier to include code examples ([like so](https://github.com/rails/rails/blob/2675c906b1cc4cc877ddee4d62dc33f544adc748/actionpack/lib/action_controller/metal/rendering.rb#L98-L134)).